### DISCUSSION TITLE
Metadata issues with two numbers

### DISCUSSION THREAD

**Discussion by valdenar** [plazi/community#81] [Q&A] [answered]
Hi Donat,
we realized that GGI only validates issues with just one number. What to do when the issue has two numbers?

We have been matching only the first number, is this the right thing to do?

![image](https://user-images.githubusercontent.com/57402751/73676971-1cc50700-4694-11ea-94d8-e6ff99bb5b63.png)

**Comment by myrmoteras** [comment]
i did the same, when I encountered this. but I will ask Guido whether there something could be done. https://github.com/plazi/Plazi-Communications/issues/991

**Comment by valdenar** [comment]
Thanks Donat!

**Comment by myrmoteras** [comment, marked as answer]
@valdenar for now just use the first number. At the end, this is not really important because we have the DOI that identifies the article unambiguously

**Comment by valdenar** [comment]
Ok Donat, thanks for your attention.
